certbot/tests/renewupdater_test.py

```diff
@@ -1,17 +1,18 @@
 """Tests for renewal updater interfaces"""
 import unittest
 
-try:
-    import mock
-except ImportError: # pragma: no cover
-    from unittest import mock
-
 from certbot import interfaces
 from certbot._internal import main
 from certbot._internal import updater
 from certbot.plugins import enhancements
 import certbot.tests.util as test_util
 
+try:
+    import mock
+except ImportError: # pragma: no cover
+    from unittest import mock
```

Comment:
Why was this not removed from #8630 (I did not go over all the changes)?

Reply:
So as @bmw said, this can be removed only once we publish a new major release version of certbot.